iset.mm

```diff
@@ -65603,6 +65603,112 @@ elements or fails to hold (is equal to ` (/) ` ) for some element.
       OXGWPVOSWQWTXASWRBWSXBXC $.
   $}
 
+  ${
+    $d A z $.  $d B w z $.  $d F w z $.  $d O w z $.  $d X w z $.  $d ph w z $.
+    fodjuomnilemdc.fo $e |- ( ph -> F : O -onto-> ( A |_| B ) ) $.
+    $( Lemma for ~ fodjuomni .  Decidability of a condition we use in various
+       lemmas.  (Contributed by Jim Kingdon, 27-Jul-2022.) $)
+    fodjuomnilemdc $p |- ( ( ph /\ X e. O )
+        -> DECID E. z e. A ( F ` X ) = ( inl ` z ) ) $=
```

Comment:
In this statement and the next one, I would use ``( ( inl |` A ) ` z )`` (since it's within a `E. z e. A`, it's equivalent). The reason is that  ``( inl |` A )`` is more tame than the raw `inl`, especially when `A` is a set. Up to you to see if it's worth it.

Reply:
I'm not sure whether to (again) try to describe my thoughts on this, or whether to just take "up to you" literally and say "I choose not to put in those restrictions".

Reply:
I thought you were still ambivalent, but if that's settled, then ok. (As you noticed, I'm often bold in my comments in order to avoid ambiguity, and apart from the occasional obvious jokes, you are encouraged to take my comments literally.)